holidays/countries/chile.py

```diff
@@ -225,6 +210,13 @@ def _get_movable(hol_date: date) -> date:
                 20,
             )
 
+    @staticmethod
+    def _summer_solstice_date(year) -> date:
+        day = 20
+        if (year % 4 > 1 and year <= 2046) or (year % 4 > 2 and year <= 2075):
+            day = 21
+        return date(year, JUN, day)
+
```

Comment:
Would look cleaner:
```suggestion
    @property
    def _summer_solstice_date(self) -> date:
        return date(
            self._year,
            JUN,
            21
            if (self._year % 4 > 1 and self._year <= 2046)
            or (self._year % 4 > 2 and self._year <= 2075)
            else 20,
        )

```

I'd also revisit the date() construction as `_add_holiday()` can be used with name, month, day args.